packages/marko-web-theme-monorail/browser/index.js

```diff
@@ -28,24 +28,6 @@ const setP1EventsIdentity = ({ p1events, brandKey, encryptedId }) => {
   p1events('setIdentity', `omeda.${brandKey}.customer*${encryptedId}~encrypted`);
 };
 
-const dispatchP1EAuthenticate = (args) => {
-  const {
-    loginSource: actionSource,
```

Comment:
@solocommand does the loginSource: actionSource setting need to be accounted for when moving this up into the list of conversion events?

Reply:
No, after review `source`, `loginSource` and `actionSource` are all set to the same value by the event emitter mixin, and `actionSource` is the preferred field for consistency. Good catch though!